app/code/Magento/Eav/Model/Attribute/Data/File.php

```diff
@@ -146,7 +146,7 @@ protected function _validateByRules($value)
             return $this->_fileValidator->getMessages();
         }
 
-        if (!empty($value['tmp_name']) && !is_uploaded_file($value['tmp_name'])) {
+        if (empty($value['tmp_name']) && !is_uploaded_file($value['tmp_name'])) {
```

Comment:
Will this ever evaluate to true?

Reply:
Yes on event where empty value of ['tmp_name'] and file not  uploaded